docs/user-guide/Links.md

```diff
@@ -0,0 +1,36 @@
+# Using OpenAPI v3 links to define dependencies
+
+The OpenAPI v3 specification defines [link objects][] that can be used to define dependencies between operations.
+Restler will use links when present in the API definition to generate dependencies between operations.
```

Comment:
>Restler will use links [](http://example.com/codeflow?start=0&length=22)

If possible, it would be great to put a short paragraph at the bottom on the supported (or not supported) subset of  Links that can be expected to work with RESTler

Reply:
In particular that only strings are supported, and operationRef and runtime expressinos are not supported

Reply:
Agreed, but I have added this to Links.md rather than here.